launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml

```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-  <!-- lidar param -->
+  <!-- LiDAR parameters -->
   <arg name="input/pointcloud"/>
   <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `clustering`"/>
   <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
   <arg name="use_object_filter" default="true" description="use object filter"/>
   <arg name="use_pointcloud_container" default="false" description="use pointcloud container for detection preprocessor"/>
   <arg name="container_name" default="pointcloud_container"/>
-  <arg name="score_threshold" default="0.20"/>
+  <arg name="lidar_detection_score_threshold" default="0.35"/>
 
-  <!-- radar param -->
+  <!-- Radar parameters -->
   <arg name="input/radar" default="/sensing/radar/detected_objects"/>
+  <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_position_filter.param.yaml"/>
```

Comment:
Should be this?
```suggestion
  <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_lanelet_filter.param.yaml"/>
```

Reply:
Fixed other default parameters at [fac721b](https://github.com/autowarefoundation/autoware.universe/pull/4995/commits/fac721b4051f4d1a2f7f659905b4204a4a048b48)